spk/bazarr/Makefile

```diff
@@ -0,0 +1,41 @@
+SPK_NAME = bazarr
+
+BAZARR_VERSION = 0.8.3.4
+
+SPK_VERS = $(BAZARR_VERSION)
+SPK_REV = 1
+
+SPK_ICON = src/bazarr.png
+
+REQUIRED_DSM = 6.1
+
+BUILD_DEPENDS = cross/python cross/setuptools cross/wheel
+
+BETA = 1
+
+DEPENDS = cross/$(SPK_NAME)
+
+WHEELS = 
+SPK_DEPENDS = "python>=2.7.13:git"
```

Comment:
Are you shure bazarr requires git spk package installed on the target system?
The installation instructions start with "installation of GIT" but IMHO git is used only to clone the bazarr repository.

Reply:
> 1. Does this kind of update really work when the original installation was not done by cloning the git repository?
> 2. As for now the git spk does not but git into the path. Therefore bazarr must use the full path to call git (/var/packages/git/target/bin/git)
> 3. In your start_daemon function in dsm-control.sh there is none of the updater arguments. What kind of update does this mean?
> 
> I would propose to disable (automatic?) updates and to remove git from SPK_DEPENDS.
> For advanced users a description how to enable updates would be helpful (modify the service start and provide a symlink /usr/local/bin/git)
> The ultimative solution would be to add an option in the install-wizard whether to enable the update feature - and to modify the service start so that the update parameter is only added when selected in the wizard and when git is installed
> 
> But I would prefer just do disable the integrated updates.

Technically git is added to the path at the startup of this script. But, then, it doesn't mean necessarily it will work.
@morpheus65535: I will enforce the disabling of the "autoupdate through git".